lib/seahorn/HornCex.cc

```diff
@@ -373,5 +386,62 @@ namespace seahorn
     out.keep ();
   }
 
+  static void sliceErrorTrace(Function &F, BmcTrace &trace) {
```

Comment:
Why could you not use reduceToRegion instead of re-implementing it?

It would also be good to have a slice through cut-points only as well.